changelogs/fragments/130-api-modify-builtin.yml

```
@@ -0,0 +1,6 @@
+minor_changes:
+  - api_info - new parametr ``include_builtin``
+    (https://github.com/ansible-collections/community.routeros/pull/130).
+bugfixes:
```

Since this does not affect anything since the last release, let's mark this as `trivial` so it won't show up in the rendered changelog.
```suggestion
trivial:
```